baselines/fedstar/fedstar/model.py

```diff
@@ -0,0 +1,239 @@
+import time
+import itertools
+import numpy as np
+import tensorflow as tf
+import tensorflow_addons as tfa

```

Comment:
`tensorflow-addons` is not recommended anymore since it will stop being maintained in May'24. It seems you use it to define fairly standard layers so there must be another way that does not require this package.

Reply:
Thanks for the suggestion I have removed tfa completely.